osu.Game.Rulesets.Taiko/UI/DrawableTaikoMascot.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Audio.Track;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Textures;
+using osu.Game.Beatmaps.ControlPoints;
+using osu.Game.Graphics.Containers;
+
+namespace osu.Game.Rulesets.Taiko.UI
+{
+    public sealed class DrawableTaikoMascot : BeatSyncedContainer
+    {
+        private static TaikoMascotTextureAnimation idleDrawable, clearDrawable, kiaiDrawable, failDrawable;
```

Comment:
why is this static? now you can't create multiple instances of this component with maintaining all of them together. (there is already an obvious existing case for this)